tracer/test/snapshots/Security.AspNetMvc5.Classic.enableSecurity=True.__test=server.response.headers.no_cookies_url=_Home_LangHeader_body=null.verified.txt

```diff
@@ -0,0 +1,272 @@
+﻿[
```

Comment:
You would expect this file to contain data about WAF rules triggering, but we can't access the headers in classic mode, so we can't detect the threads in this case.

Reply:
good point about the classic mode!.. could we add a comment on the related test so that we remember? (I should so that I remember 😅 )

Reply:
Added a comment to the test in 0ce2bcc51